services/Google.filters.js

```diff
@@ -4,3 +4,23 @@ export function removeUTMfromUrls(document) {
     link.href = link.href.replace(/utm_[^=]*=[^&]*[&]{0,1}/g, '');
   });
 }
+
+export function removeCountryVersion(document) {
+  const countryVersionLink = document.querySelector('a[data-name="country-version"]');
+  const countryVersionTitle = document.querySelector('#footnote-country-version');
+  const disclaimerTitle = document.querySelector('#footnote-disclaimer');
+
+  console.log('countryVersionLink', countryVersionLink);
+  if (countryVersionLink) {
+    // Remove "country version" link at top of the page
+    countryVersionLink.parentNode.remove();
+  }
+
+  if (countryVersionTitle && disclaimerTitle) {
+    // Remove all the "country version" section in the CGUs
+    const countryVersionSectionContent = document.createRange();
+    countryVersionSectionContent.setStartBefore(countryVersionTitle);
```

Comment:
Proposal:

```js
console.assert(countryVersionSectionContent.cloneContents().querySelectorAll('h3').length === 1, 'Country version selector seems to be too large.');
```

Reply:
What about directly throwing an error to avoid recording a incorrect version? :)

Reply:
:+1: